src/core/Akka.Streams.Tests/Dsl/FlowSplitAfterSpec.cs

```diff
@@ -86,112 +87,115 @@ public StreamPuppet(IPublisher<int> p, TestKitBase kit)
         }
 
         [Fact]
-        public void SplitAfter_must_work_in_the_happy_case()
+        public async Task SplitAfter_must_work_in_the_happy_case()
         {
-            this.AssertAllStagesStopped(() =>
-            {
-                WithSubstreamsSupport(3,5,run: (masterSubscriber, masterSubscription, expectSubFlow) =>
-                {
-                    var s1 = new StreamPuppet(expectSubFlow().RunWith(Sink.AsPublisher<int>(false), Materializer), this);
-                    masterSubscriber.ExpectNoMsg(TimeSpan.FromMilliseconds(100));
-
-                    s1.Request(2);
-                    s1.ExpectNext(1);
-                    s1.ExpectNext(2);
-                    s1.Request(1);
-                    s1.ExpectNext(3);
-                    s1.Request(1);
-                    s1.ExpectComplete();
-
-                    var s2 = new StreamPuppet(expectSubFlow().RunWith(Sink.AsPublisher<int>(false), Materializer), this);
-                    s2.Request(2);
-                    s2.ExpectNext(4);
-                    s2.ExpectNext(5);
-                    s2.ExpectComplete();
-
-                    masterSubscription.Request(1);
-                    masterSubscriber.ExpectComplete();
-                });
+            await this.AssertAllStagesStoppedAsync(() => {
+                WithSubstreamsSupport(3, 5, 
+                    run: (masterSubscriber, masterSubscription, expectSubFlow) =>                                                                         
+                    {                                                                             
+                        var s1 = new StreamPuppet(expectSubFlow()
+                            .RunWith(Sink.AsPublisher<int>(false), Materializer), this);                                                                             
+                        masterSubscriber.ExpectNoMsg(TimeSpan.FromMilliseconds(100));
+                                                                             
+                        s1.Request(2);                                                                             
+                        s1.ExpectNext(1);                                                                             
```

Comment:
Shouldn't these methods be made `async`? Aren't there already `async` alternatives in the TestKit?